sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptedBlobRange.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Http;
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// This is a representation of a range of bytes on an encrypted blob, which may be expanded from the requested range to
+    /// included extra data needed for encryption.Note that this type is not strictly thread-safe as the download method
+    /// will update the count in case the user did not specify one. Passing null as an EncryptedBlobRange value will default
```

Comment:
> the download method will update the count in case the user did not specify one

Where is this happening?  I think I lost it in the PR.  Does it muck directly with the value?  Could we add an `UpdateRangeLength` method or something to make it a little more clear?

Reply:
I think the docstring is no longer accurate. I copy/pasted from Java's code before modifying the code. Will clarify.